.github/workflows/automatic-release.yml

```diff
@@ -57,5 +63,5 @@ jobs:
 
       - name: Release
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.USER_TOKEN != '' && secrets.USER_TOKEN || secrets.GITHUB_TOKEN }}
```

Comment:
not sure about it, to be more accurate should be: READ_WRITE_TOKEN, and what about the different areas mentioned here? https://github.com/semantic-release/github#github-authentication 

Maybe **GH_AUTH_TOKEN** ?

Reply:
Regarding the proposed logic, I am not adjusting to the proposed because the current one is leveraging the weird ternary operator from GH actions: see: https://docs.github.com/en/actions/learn-github-actions/expressions#example 

Current logic explanation:
If USER_TOKEN is not empty then return me that value, else the GITHUB_TOKEN

Proposed logic explanation:
(secrets.USER_TOKEN && secrets.USER_TOKEN != '') -> since the second part is an expression and not a value I think GH will resolve to a boolean value if the result is false will return GITHUB_TOKEN. The other scenario is -> if USER_TOKEN is truthy then it will return which means the expression `secrets.USER_TOKEN != ''` is not considered. 

Am I missing something?